File: flutter_app/lib/src/meeting/models/meeting_event.dart
class MeetingEvent {
  String? room;
  String? msg;
  String? sender;
  String? to;
  MeetingEvent({this.room, this.msg, this.sender, this.to});

  MeetingEvent.fromJson(Map<String, dynamic> json)
      : room = json['room'],
        msg = json['msg'],
        sender = json["sender"],
        to = json["to"];

  Map<String, dynamic> toJson() {
    return {
      'room': room,
      'msg': msg,
      'sender': sender,
      'to': to,
    };
  }

  @override
  String toString() {
    return 'MeetingEvent{room: $room, msg: $msg, sender: $sender, to: $to}';
  }
}

File: backend/src/main/java/com/example/backend/meeting/socketio/MeetingEvent.java
package com.example.backend.meeting.socketio;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class MeetingEvent {
    String room;
    String msg;
    String sender;
    String to;
}
